tests/test_user.py

```diff
@@ -218,6 +217,48 @@ def test_noop(self, requests_mock, cli_runner):
         cli_runner.invoke(change_command, ["1"])
         assert not requests_mock.called
 
+    @pytest.mark.usefixtures("mock_change_password_url")
+    def test_success_change_password(self, requests_mock, cli_runner):
+        new_password = "new_password"
+        data = {
+            "new_password1": new_password,
+            "new_password2": new_password,
+            "old_password": "old_password",
+        }
+        user_id = "1"
```

Comment:
Other tests are using user ID as an integer. Based on our API docs, the user id is an integer, not a string.

Reply:
Please fix this for all the tests.

Reply:
should I change it only for tests in `test_user.py`?

Reply:
Change it in all tests where in reality `int` is passed instead of `str`.